### PR TITLE
chore: Improve descriptions per code review

### PR DESCRIPTION
**Pull request by gibson042**
Stacked on #876

### CODE REVIEW

**Comment by erights** [comment]
@gibson042 , since it says "Stacked on #876" I changed the github comparison base to reflect that. Makes a lot more sense to me now.

File: main/glossary/README.md [hunk 1]
```diff
@@ -554,13 +554,11 @@ it set at $10. They can specify the instance's minimum bid amount in its terms.
 
 ## Value
```
Comment (erights) [inline main/glossary/README.md:555]:
```suggestion
## AmountValue
```

File: main/reference/ertp-api/ertp-data-types.md [hunk 2]
```diff
@@ -36,12 +36,12 @@ someAmountShape: {
 
 ## Value
```
Comment (erights) [inline main/reference/ertp-api/ertp-data-types.md:37]:
```suggestion
## AmountValue
```

**Comment by gibson042** [comment]
> I also saw your related question on the base PR. The problem is that we use the term "value" in two contradictory ways: As the value part of an amount, or as the value of a map's key-value mapping. The unqualified term "value" should only refer to the latter. For the value part of an amount, the actual name of the type in the code is already `AmountValue`.
> 
> With this straightened out, any AmountValue is a Key. A value of a CopyMap's key-value mapping may not be a Key.

Done, but it does introduce some minor reading friction: https://github.com/Agoric/documentation/pull/886/commits/fa576aece9f13254e4a4f0625b10d7fba0fc404d

File: main/glossary/README.md (hunk 1 above)
Comment (erights) [reply main/glossary/README.md]: Seems done. Resolving.

File: main/reference/ertp-api/ertp-data-types.md (hunk 2 above)
Comment (erights) [reply main/reference/ertp-api/ertp-data-types.md]: Seems done. Resolving.

**Pull request merged**